index.html

```diff
@@ -14,7 +14,7 @@
 
         <div class="page-heading">How Istanbul works</div>
         <p>
-          Istanbul instruments your ES5 and ES6 JavaScript code with line counters,
+          Istanbul instruments your ES5 and ES2016+ JavaScript code with line counters,
```

Comment:
Right, so this is effectively saying: "Istanbul instruments your ES5 and ES2016+, but doesn't do anything with ES2015 features"